src/Widgets/DataLabelWidget/DataLabelWidgetEditingConfig.ts

```diff
@@ -28,6 +28,26 @@ provideEditingConfig(DataLabelWidget, {
     marginBottom: {
       title: 'Add margin bottom?',
     },
+    valueSize: {
+      title: 'Value size',
+      description: 'Size and font of this value. Default: Body font size',
```

Comment:
* [ ] This is confusing. It's unclear what "this value" refers to. Many data labels don't show a value but just text. Not sure if mentioning the font makes sense b/c it can't really be chosen here but can (only) be set in the site settings. Maybe we should make it similar to the headline widget: "Style" and "Size of the label".
* [ ] FYI not introduced here but the headline widget also mentions "font".